src/index.ts

```diff
@@ -88,14 +88,21 @@ extension.addBuildEventHandler(
     }
 
     if (!tempConfig) {
-      config = {
-        reportOnly: true,
-        reportUri: "",
-        unsafeEval: true,
-        path: ["/*"],
-        excludedPath: [],
-      };
-      console.log("Using default CSP config.");
+      // FIXME(ndhoule): This is a bug. At one point we added a default value for config, which
+      // means we can no longer enter this case.
+      // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
+      if (!config) {
```

Comment:
The bug here is: we default config to `{}` where it's defined, so this condition is impossible. But I spaced and made the CSP extension _always_ use the default config.

(I have a branch that will fix various issues in the CSP extension and add some tests, so I'll add a regression test for this.)